src/lib/datepicker/multi-year-view.ts

```diff
@@ -101,6 +101,9 @@ export class MatMultiYearView<D> implements AfterContentInit {
   /** Emits the selected year. This doesn't imply a change on the selected date */
   @Output() readonly yearSelected: EventEmitter<D> = new EventEmitter<D>();
 
+  /** Emits when any date is activated. */
+  @Output() readonly activeDateChange: EventEmitter<D> = new EventEmitter<D>();
```

Comment:
Add unit test for these new outputs?

Reply:
Updated the tests to work the same way as month-view tests, which test the `@Output()`